src/types/aws/ap/registration.ts

```diff
@@ -108,6 +115,7 @@ export type FSASignerDocumentation = {
 export type NonFSADocumentation = {
   DocType: "Non-FSA";
   EIN: string;
+  Claim?: EndowClaim | null;
```

Comment:
Can this be both `null` and `undefined`? Maybe one of them is redundant?

Reply:
`null` represents `Claim` being reset in the server e.g.
fresh (no claim) -> incidentally inputs unclaimed `EIN` (setting claim) -> enters new EIN instead (resets the `Claim: null`)